.github/workflows/deploy-landingpage-new-on-change.yml

```diff
@@ -0,0 +1,52 @@
+name: Deploy Landingpage New On Change
+
+on:
+  push:
+    branches: 
+      - master
+    paths:
+      - 'packages/landingpage-new/**'
```

Comment:
The actions are still the same as in `deploy-new-landingpage` right? If that's the case, it might be sufficient to move the trigger (the stuff I highlighted) to the `deploy-new-landingpage` action. I think you can define many triggers for one action